src/openforms/registrations/tests/test_registration_hook.py

```diff
@@ -47,6 +47,7 @@ def setUpTestData(cls):
 
         cls.submission = SubmissionFactory.create(
             completed=True,
+            pre_registration_completed=True,
```

Comment:
imo this `pre_registration_completed` should be made part of the `completed` trait.